kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1503,6 +1503,14 @@ object KyuubiConf {
       .checkValue(_ > 0, "the maximum must be positive integer.")
       .createWithDefault(10)
 
+  val SESSION_ENGINE_DESTROY_TIMEOUT: ConfigEntry[Long] =
+    buildConf("kyuubi.session.engine.destroy.timeout")
+      .doc("Timeout(ms) for the engine destroy,if engine destroy times out," +
+        "force destroy instead.")
```

Comment:
Suggest move the definition after `buildConf("kyuubi.session.engine.startup.waitCompletion")`

```suggestion
      .doc("Engine startup process destroy wait time, if the process does not " +
         "stop after this time, force destroy instead. This configuration only " +
         s"takes effect when `${SESSION_ENGINE_STARTUP_WAIT_COMPLETION.key}=false`.")
```

Reply:
missed the comments?

Reply:
ok